addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18813,7 +18819,23 @@ msgctxt "#36050"
 msgid "On start"
 msgstr ""
 
-#empty strings from id 36051 to 36097
+#. Item list value of setting with label #36013 "Represent type of device"
```

Comment:
#. Item list value of setting with label #36013 "CEC client device mode"

And I guess you copy/paste this three times for all values? Maybe the description should include info about the item?